obj/Debug/net8.0/.NETCoreApp,Version=v8.0.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
```

Comment:
Debug files should have been added to .gitignore and not versioned

Reply:
Why ?